_posts/2016-11-05-Publications.md

```diff
@@ -25,31 +25,41 @@ This is a list of some works that contributed to JuPedSim or used it to
 produce results.
 
 ## Theses
-- Benjamin Schröder
+- Hein, Ben
+  [Agent-based modelling for crowding and queuing in front of bottlenecks]
+  Master Thesis, University of Wuppertal, 2019
```

Comment:
you have a link?

Reply:
i will upload it to sciebo and add the link